src/main/java/org/traccar/protocol/LaipacProtocolEncoder.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2015 - 2019 Anton Tananaev (anton@traccar.org)
```

Comment:
This should be updated.

Reply:
Are you suggesting that I use my name there?

Reply:
You can use your name if you want, but I'm referring to the years.

Reply:
Oh, I see, this is a new file. Sorry, I'll fix it. Thanks.